### PR TITLE
chore(Custom Status): metadata update

### PR DESCRIPTION
**Pull request by Hadlex**
**Describe the changes in this pull request:**
Updated the description, the presence now support image customization. "name" can be "title" or "presence name", but that's how it's called in the developer portal.

### CODE REVIEW

File: websites/C/Custom Status/dist/metadata.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	},
 	"service": "Custom Status",
 	"description": {
-		"en": "Custom Status is a way to set custom statuses using PreMiD. You can customize everything but using custom images. You have to visit [this page](https://eggsy.xyz/projects/premid/custom-status) to start using!",
+		"en": "Custom Status is a way to set custom statuses using PreMiD. You can customize everything except the name. You have to visit [this page](https://eggsy.xyz/projects/premid/custom-status) to start using it!",
 		"tr": "Custom Status, Discord'un kendi özelliği gibi isteğinize göre ayarlanabilen bir oynuyor kısmıdır. Bu servis ile PreMiD kullanarak istediğiniz resmi seçme olanağı dışında bir RPC ile yapabileceğiniz her şeyi yapabilirsiniz. Servisin çalışması için servisi ekleyip, [buradaki](https://eggsy.xyz/projects/premid/custom-status) sayfaya gitmelisiniz.",
```
Comment (EncryptedDev) [inline websites/C/Custom Status/dist/metadata.json:10]: @eggsy can we have a translation here?

**Comment by EncryptedDev** [comment]
Waiting for the translations before merging 👍🏼

File: websites/C/Custom Status/dist/metadata.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
 		"nl": "Custom Status staat je toe aangepaste statussen weer te geven met PreMiD. Je kunt alles aanpassen behalve het gebruik van eigen afbeeldingen. Bezoek [deze pagina](https://eggsy.xyz/projects/premid/custom-status) om te beginnen!",
```
Comment (EncryptedDev) [inline websites/C/Custom Status/dist/metadata.json:11]: @QkeleQ10 or @Bas950 can you translate the changed string?

File: websites/C/Custom Status/dist/metadata.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
...
 		"vi_VN": "Custom Status là một cách để bạn đặt trạng thái tuỳ chọn qua PreMiD. Bạn có thể tuỳ chỉnh tất cả mọi thứ. Bạn phải vào [đường dẫn này](https://eggsy.xyz/projects/premid/custom-status) để bắt đầu sử dụng!"
```
Comment (EncryptedDev) [inline websites/C/Custom Status/dist/metadata.json:12]: @kyrie25 can you translate the changed string?

Comment (kyrie25) [reply websites/C/Custom Status/dist/metadata.json:12]: > @kyrie25 can you translate the changed string?

It was actually translated for this description (no need to change)

File: websites/C/Custom Status/dist/metadata.json (hunk 2 above)
Comment (joerkig) [reply websites/C/Custom Status/dist/metadata.json:11]:
```suggestion
		"nl": "Custom Status staat je toe aangepaste statussen weer te geven met PreMiD. Je kunt alles aanpassen behalve de naam. Bezoek [deze pagina](https://eggsy.xyz/projects/premid/custom-status) om te beginnen!",
```

File: websites/C/Custom Status/dist/metadata.json (hunk 1 above)
Comment (Hadlex) [reply websites/C/Custom Status/dist/metadata.json:10]: Eggsy told me to do this change so i don't think he can.

Comment (EncryptedDev) [reply websites/C/Custom Status/dist/metadata.json:10]: EGGSY sent this to me
```suggestion
"tr": "Custom Status, Discord’un kendi özelliği gibi isteğinize göre ayarlanabilen bir oynuyor kısmıdır. Bu servis ile PreMiD kullanarak bir RPC ile yapabileceğiniz her şeyi yapabilirsiniz. Servisin çalışması için servisi ekleyip, [buradaki](https://eggsy.xyz/projects/premid/custom-status) adrese gitmelisiniz.",
```

File: websites/C/Custom Status/dist/metadata.json (hunk 3 above)
Comment (EncryptedDev) [reply websites/C/Custom Status/dist/metadata.json:12]: Awesome, thank you!

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**